### ISSUE TITLE
Error compiling py3 branch of pyodbc

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maximemaher/pyodbc#206]
```
I've downloaded py3 branch of pyodbc:

# git clone --branch py3 https://code.google.com/p/pyodbc/

After I've made compiling:

# cd pyodbc
# python3 setup.py install

But doesn't compile due an C error:

  cc1plus: warning: command line option "-Wstrict-prototypes" is valid     for Ada/C/ObjC but not for C++
  In file included from /tmp/pyodbc/src/pyodbc.h:151:0,
                 from /tmp/pyodbc/src/row.cpp:12:
  /tmp/pyodbc/src/pyodbccompat.h:106:8: warning: extra tokens at end of #endif directive
  /tmp/pyodbc/src/row.cpp:386:1: error: invalid conversion from ‘PyObject* (*)(PyObject*, Py_ssize_t, Py_ssize_t)’ to ‘void*’
  /usr/include/python3.2mu/datetime.h:185:25: warning: ‘PyDateTimeAPI’  defined but not used
  error: command 'gcc' failed with exit status 1








```

Original issue reported on code.google.com by `andres.o...@gmail.com` on 30 Aug 2011 at 1:24

**Comment by GoogleCodeExporter** [comment]
```
Issue 201 is stuck on the same thing:
http://code.google.com/p/pyodbc/issues/detail?id=201&sort=-id
```

Original comment by `nickl...@gmail.com` on 31 Aug 2011 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Ok. I though that in Isse 201 was compiling with python 2.7 (2. Build using 
python 2.7 (setyp.py doesn't seem to work in python 3.x, syntax error)).

Sorry.
```

Original comment by `andres.o...@gmail.com` on 1 Sep 2011 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
You are right, I just wanted to say "me too", really.
```

Original comment by `nickl...@gmail.com` on 1 Sep 2011 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
Same here Python 3.2.2 64 bit Ubuntu 11.04
```

Original comment by `neilbed...@gmail.com` on 27 Sep 2011 at 11:13

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.0.1

```

Original comment by `mkleehammer` on 3 Nov 2011 at 1:50
- Changed state: **Complete**
